x-pack/plugins/triggers_actions_ui/public/application/lib/breadcrumb.ts

```diff
@@ -6,41 +6,47 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { routeToHome, routeToConnectors, routeToRules, routeToRuleDetails } from '../constants';
+import { routeToHome, routeToConnectors, routeToRules } from '../constants';
 
-export const getAlertingSectionBreadcrumb = (type: string): { text: string; href: string } => {
+export const getAlertingSectionBreadcrumb = (
+  type: string,
+  returnHref: boolean = false
+): { text: string; href?: string } => {
   // Home and sections
   switch (type) {
     case 'connectors':
       return {
         text: i18n.translate('xpack.triggersActionsUI.connectors.breadcrumbTitle', {
           defaultMessage: 'Connectors',
         }),
-        href: `${routeToConnectors}`,
+        ...(returnHref
+          ? {
+              href: `${routeToConnectors}`,
+            }
+          : {}),
       };
     case 'rules':
       return {
         text: i18n.translate('xpack.triggersActionsUI.rules.breadcrumbTitle', {
           defaultMessage: 'Rules',
         }),
-        href: `${routeToRules}`,
+        ...(returnHref
+          ? {
+              href: `${routeToRules}`,
+            }
+          : {}),
       };
     default:
       return {
         text: i18n.translate('xpack.triggersActionsUI.home.breadcrumbTitle', {
           defaultMessage: 'Rules and Connectors',
         }),
         href: `${routeToHome}`,
```

Comment:
should this `href` field be removed?

Reply:
Oh yes, thanks for catching that!

Reply:
Resolved in https://github.com/elastic/kibana/pull/141838/commits/02c7d0402471cdf3f8d5e3f160fd7a9dda46ccf5